__tests__/app.test.js

```diff
@@ -68,43 +68,67 @@ describe("/api", () => {
         });
     });
   });
-});
-
-describe("PATCH /api/articles/:article_id ", () => {
-  test("201: returns an object with updated votes property by the provided number", () => {
-    const req = { inc_votes: 3 };
-    const expected = {
-      article_id: 1,
-      title: "Living in the shadow of a great man",
-      topic: "mitch",
-      author: "butter_bridge",
-      body: "I find this existence challenging",
-      created_at: expect.any(Date),
-      votes: 103,
-    };
-    return request(app)
-      .patch("/api/articles/1")
-      .send(req)
-      .expect(201)
-      .then(({ body }) => {
-        const formattedResponse = convertTimestampToDate(body.updatedVotes);
-        expect(formattedResponse).toEqual(expected);
-      });
-  });
-  test("404: Returns the not found message when passed article id that doesn't exist", () => {
-    return request(app)
-      .get("/api/articles/99999999")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toBe("not found");
-      });
+  describe("PATCH /api/articles/:article_id ", () => {
+    test("201: Returns an object with updated votes property by the provided number", () => {
```

Comment:
This would be 200 - you've not created something, just amended.

Reply:
Yes, indeed. Thank you!